### DISCUSSION TITLE
Corrupted image while training in YOLOv5

### DISCUSSION THREAD

**Discussion by kashishgoyal31** [ultralytics/yolov5#4525] [Q&A] [answered]
When I am training on custom dataset, the model is not accepting the images and prompts the error of corrupted image, sharing screenshot for same. Please provide the solution as 90% of the dataset is not considered. 

![corruptedimageerror](https://user-images.githubusercontent.com/89016389/130565810-f6ec550e-dd4d-437e-af7f-fcc2c95a1de9.PNG)

**Comment by glenn-jocher** [comment, marked as answer]
@kashishgoyal31 this indicates that your jpegs have corruptions issues. You can resolve this by loading and saving these each with cv2 for example. We don't do this automatically because this is an indicator that your jpegs may be incomplete or have serious problems.

**Comment by kashishgoyal31** [comment]
Sir I have around 3000 images and all arr of similar quality and clicked
with same directiona. While training few images are being read and rest
it's showing corrupted. Also the image is getting opened in windows without
any error message.


Regards
Kashish Goyal
